### PR TITLE
Add the support of JSON.MGET command

### LINKED ISSUE #1816: Add support for the JSON.MGET command
Please refer to https://github.com/apache/kvrocks/issues/1805.

### PR DESCRIPTION
**Pull request by skyitachi**
Closes #1816.

### CODE REVIEW

File: src/types/redis_json.cc [hunk 1]
```diff
@@ -512,4 +512,85 @@ rocksdb::Status Json::ObjLen(const std::string &user_key, const std::string &pat
...
+      auto res = JsonValue::FromString(raw_values[i]);
+      if (!res) {
+        statuses[i] = rocksdb::Status::Corruption(res.Msg());
+        continue;
+      }
+      values[i] = *std::move(res);
+    } else if (meta_data[i].format == JsonStorageFormat::CBOR) {
+      auto res = JsonValue::FromCBOR(raw_values[i]);
+      if (!res) {
+        rocksdb::Status::Corruption(res.Msg());
```
Comment (PragmaTwice) [inline src/types/redis_json.cc]:
```suggestion
        statuses[i] = rocksdb::Status::Corruption(res.Msg());
```

File: src/types/redis_json.cc [hunk 2]
```diff
@@ -512,4 +512,85 @@ rocksdb::Status Json::ObjLen(const std::string &user_key, const std::string &pat
...
+        continue;
+      }
```
Comment (PragmaTwice) [inline src/types/redis_json.cc:568]:
```suggestion
      } else {
        ... // for unknow format
      }
```

File: src/types/redis_json.cc (hunk 1 above)
Comment (skyitachi) [reply src/types/redis_json.cc]: fixed

**Review by git-hulk — APPROVED** [review]
LGTM

**Pull request merged**